style.css

```diff
@@ -0,0 +1,395 @@
+
+.nav-bar a {
+	text-decoration: none;
+	color: #000000;
+}
+
+.top-bar .hamburger {
+	display: none;
+}
+
+main {
+	background-color:#474c58;
+	color: #ffffff;
+	padding: 5%;
+}
+
+.homepage {
+	width: 100%;
+	height: 800px;
+}
+
+.titlepage {
+	font-family: 'Montserrat', sans-serif;
+	margin: 5%;
+	font-size: 1.75rem;
+	position: relative;
+}
+
+.title-format {
+	display: flex;
+	margin: 25px;
+}
+
+.title-letter {
+	border: 4px solid #ffffff;
+	padding: 17px;
+	margin: 20px;
+	height: 75px;
+	width: 75px;
+	text-align: center;
+	color: #000000;
+}
+
+#title-letter-1 {
+	background-color: #736a87;
+}
+
+#title-letter-1:hover {
+	background-color: #e6f0f5;
+}
+
+#title-letter-2 {
+	background-color: #9a8eb4;
+}
+
+#title-letter-2:hover {
+	background-color: #cee1ea;
+}
+
+#title-letter-3 {
+	background-color: #c0b1e1;
+}
+
+#title-letter-3:hover {
+	background-color: #b5d2e0;
+}
+
+#title-letter-4 {
+	background-color: #cdc1e7;
+}
+
+#title-letter-4:hover {
+	background-color: #8aa4b0;
+}
+
+#title-letter-5 {
+	background-color: #d9d0ed;
+}
+
+#title-letter-5:hover {
+	background-color: #79909a;
+}
+
+#title-letter-6 {
+	background-color: #e6e0f3;
+}
+
+#title-letter-6:hover {
+	background-color: #57676e;
+}
+
+#title-letter-7 {
+	background-color: #6a7e87;
+}
+
+#title-letter-7:hover {
+	background-color: #e0e1f3;
+}
+
+#title-letter-8 {
+	background-color: #8ea8b4;
+}
+
+#title-letter-8:hover {
+	background-color: #b1b5e1;
+}
+
+#title-letter-9 {
+	background-color: #b1d2e1;
+}
+
+#title-letter-9:hover {
+	background-color: #8e91b4;
+}
+
+#title-letter-10 {
+	background-color: #c8e0ea;
+}
+
+#title-letter-10:hover {
+	background-color: #7c7f9e;
+}
+
+.video-col video {
+	position: absolute;
+	right: 15%;
+	top: 30%;
+	width: 400px;
+	height: 350px;
+	z-index: 0;
+	opacity: 0.8;
+}
+
+.portfolio-link {
+	width: 150px;
+	height: 110px;
+	background: #c0b1e1;
+	text-decoration: none;
+	color: #000000;
+	font-size: 1.75rem;
+	font-family: 'Montserrat', sans-serif;
+	padding: 70px 35px 90px 17px;
+	border: 2px solid #ffffff;
+	position: absolute;
+	right: 20%;
+	top: 30%;
+	z-index: 1;
+}
+
+.portfolio-link:hover {
+	background-color: #d2e1b1;
+}
+
+.about-container {
+	display: flex;
+	justify-content: space-evenly;
+	gap: 5%;
+}
+
+.saskia-image {
+	width: 400px;
+}
+
+.portfolio {
+	display: grid;
+	grid-template-columns: 1fr 1fr 1fr;
+	gap: 20px;
+	background-color: #474c58;
+	margin: 1%%;
+}
+
+.portfolio .container {
+	margin: auto;
+	width: 270px;
+	height: 270px;
+	background: #c0b1e1;
+	border: 5px solid #c5e0dd;
+	padding: 7px;
+	text-align: center;
+	font-size: 0.85rem;
+}
+
+.container .title-project {
+	font-size: 1.1rem;
+	text-align: center;
+	
+}
+
+.contact-section {
+	display: flex;
+	flex-direction: wrap;
+}
+
+.contact-col {
+	margin: 10px;
+	padding: 10px;
+	border: 2px solid #ffffff;
+	width: 400px;
+	background-color: #c0b1e1;
+	align-content: center;
+	
+}
+
+.image-contact {
+	width: 400px;
+	margin: 10px;
+}
+
+footer {
```

Comment:
you created a class footer, but I think you didn't use it, have a look if you need it or just want to keep targeting the footer (no dot - no class)